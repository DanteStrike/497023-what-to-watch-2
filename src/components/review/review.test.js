import React from "react";
import renderer from "react-test-renderer";

import Review from "./review.jsx";


it(`Render correctly Review component`, () => {
  const component = renderer
    .create(
        <Review
          author={`author`}
          rating={10}
          comment={`any`}
          date={0}
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
