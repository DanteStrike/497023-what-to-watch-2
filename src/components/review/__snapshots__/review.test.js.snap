// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly Review component 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      any
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        author
      </cite>
      <time
        className="review__date"
        dateTime="1970-01-01"
      >
        January 01, 1970
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    10
  </div>
</div>
`;
